src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -289,7 +289,7 @@
     <value>Expected a digit ('0'-'9'), but instead reached end of data.</value>
   </data>
   <data name="SpecialNumberValuesNotSupported" xml:space="preserve">
-    <value>.NET number values such as positive and negative infinity cannot be written as valid JSON.</value>
+    <value>.NET number values such as positive and negative infinity cannot be written as valid JSON. Consider JsonNumberHandling.AllowNamedFloatingPointLiterals to make it work (see https://docs.microsoft.com/dotnet/api/system.text.json.serialization.jsonnumberhandling).</value>
```

Comment:
- Quote parts of the message that we don't want to be translated/localized in the future.
- Note that the workaround will only applicable when using 'JsonSerializer' (and not the writer directly - https://dotnetfiddle.net/kFCdTs).


```suggestion
    <value>.NET number values such as positive and negative infinity cannot be written as valid JSON. To make it work when using 'JsonSerializer', consider specifying 'JsonNumberHandling.AllowNamedFloatingPointLiterals' (see https://docs.microsoft.com/dotnet/api/system.text.json.serialization.jsonnumberhandling).</value>
```